e2e/api/setup.js

```diff
@@ -1,16 +1,18 @@
 import { assert } from 'chai';
 import config from 'config';
-import dbClient from '../../src/api/lib/db/client';
+import { PgPool } from 'co-postgres-queries';
 import fixturesFactory from '../lib/fixturesLoader';
 import request from '../lib/request';
 
 before(function* setGlobalForTest() {
-    global.db = yield dbClient(config.apps.api.db);
-    global.fixtureLoader = fixturesFactory(global.db.client);
+    global.pool = new PgPool(config.apps.api.db);
```

Comment:
We had a talk with @djhi @fzaninotto @alexisjanvier about the best way to share variables... this is another way :)